name.abuchen.portfolio/src/name/abuchen/portfolio/snapshot/ReportingPeriod.java

```diff
@@ -56,6 +56,11 @@ private Type(char code, Class<? extends ReportingPeriod> implementation)
             this.code = code;
             this.implementation = implementation;
         }
+
+        public char getCode()
+        {
+            return code;
```

Comment:
Why this getter? The code is an implementation detail that doesn’t need to be public. (You aren’t using the getter, either.)

Reply:
Thx. I try something out and then forgot to remove it. Now it's removed.